schema/src/main/protobuf/checkpoint/checkpoint_block.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package org.constellation.schema.proto.checkpoint;
+
+import "checkpoint/checkpoint_edge.proto";
+
+message CheckpointEdgeLike {
+  CheckpointEdge checkpoint = 1;
+}
+
+message CheckpointBlockMetadata {
+  string transactionsMerkleRoot = 1;
+  CheckpointEdge checkpointEdge = 2;
+  string messagesMerkleRoot = 3;
+  string notificationsMerkleRoot = 4;
+  string observationsMerkleRoot = 5;
+
+  // from CheckpointEdgeLike
+  CheckpointEdge checkpoint = 6;
+}
+
+message CheckpointBlock {
+//  repeated Transaction = 1;
+  CheckpointEdge checkpoint = 2;
```

Comment:
Will that work? Assigning 2 straight away and commenting out fields?

Reply:
yes but I can actually uncomment it now